react-frontend/src/components/CoCos/coCoCard.js

```diff
@@ -5,9 +5,10 @@ import { Link } from 'react-router-dom';
 
 import { Badge, BadgeWrapper } from '../StyleComponents/badge';
 import {
-  CardTitle,
-  CardDescription,
-  CardStyled,
+  CommonComponentCardDescription,
```

Comment:
Please change the naming convention to CoCoCardDescription etc.  This will match the existing names for common component files.

Reply:
Right, thanks. I've reverted back in:  https://github.com/bcgov/digital.gov.bc.ca/pull/869/commits/90fa2f19e03d8c6285848bb88162847e6f382a11